libraries/nexus-schema/src/server.ts

```diff
@@ -1,15 +1,17 @@
 import express from 'express'
 import { graphqlHTTP } from 'express-graphql'
+import { createContext } from './context'
 import { schema } from './schema'
 
 const app = express()
 
 app.use(
   '/',
-  graphqlHTTP({
+  graphqlHTTP((req) => ({
     schema,
+    context: createContext(),
```

Comment:
Any idea why this is needed @jasonkuhrt? The examples on https://github.com/graphql/express-graphql#simple-setup don't use it.

Reply:
That's the way resolver context gets created for every request. If a developer does not provide it, then express-graphql defaults to making the resolver context be the request object.

Reply:
https://github.com/graphql/express-graphql/issues/187

Reply:
Why does the default not work for our project?

Reply:
Because we need to expose prisma client on the resolver context.

I don't think we need a context function though. That's just how it was setup before.